sds_data_manager/lambda_code/SDSCode/rds_utils/db_ingest_query.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import sys
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+
+class DbIngestQuery:
+    """
+    Class to represent the connection to the ingest query.
```

Comment:
If I'm following this right, this class actually constructs the query, and then allows itself to be filled, but doesn't do any DB connections, that is handled in your other `DbConnection` class right? If that is the case, I wonder if this would make more sense as a dataclass or similar. So this class would create the query information to be submitted to psycopg (which you already have here), but then it would be more of a function call to `records = db_connection.send_query(DbIngestQuery)` that would return the cursor records instead of filling a tuple you give it?